browser/js/common/directives/navbar/navbar.html

```diff
@@ -1,30 +1,78 @@
-<nav class="navbar navbar-static-top">
-    <div class="container">
-        <ul>
-            <li>
-                <a ng-click="goToSection('.problem')"> The Problem </a>
-            </li>
+<nav id='navbar' class='navbar navbar-static-top'>
+	<div class='container'>
+		<div class='dropdown'>
+			<div class='dropdown-title'>
+				What is Kinetic
+			</div>
+				<div class='dropdown-content'>
+					<div class='list-container'>
+						<a ng-click="goToSection('.problem', 'home')"> The Problem </a>
+						<a ng-click="goToSection('.solution', 'home')"> Our Solution </a>
+						<a ng-click="goToSection('.what-is', 'home')"> What We Do </a>
+						<a ng-click="goToSection('.about-us', 'home')"> About Us </a>
+						<a ng-click="goToSection('.sponsored', 'home')"> Key Issues </a>
+						<a ng-click="goToSection('.get-involved', 'home')"> Get Involved </a>
+					</div>
+				</div>
+		</div>
 
-            <li>
-                <a ng-click="goToSection('.solution')"> Our Solution </a>
-            </li>
+		<!-- TODO Add in a second parameter for all the ng-click vals for the state where each of these links should go to -->
+		<!--<div class='dropdown'>
+			<div class='dropdown-title'>
+				Get Involved
+			</div>
+				<div class='dropdown-content'>
+					<div class='list-container'>
+						<a ng-click="goToSection('.launch')"> Launch a Chapter </a>
+						<a ng-click="goToSection('.mentor')"> Become a Mentor </a>
+						<a ng-click="goToSection('.nominate')"> Nominate an Expert </a>
+						<a ng-click="goToSection('.support')"> Support Kinetic Global </a>
+					</div>
+				</div>
+		</div>-->
 
-            <li>
-                <a ng-click="goToSection('.what-is')"> What is Kinetic </a>
-            </li>
+		<div class='dropdown'>
+			<div class='dropdown-title'>
+				Resources
+			</div>
+				<div class='dropdown-content'>
+					<div class='list-container'>
+						<!--<a ng-click="goToSection('.expert')"> Expert Network </a>-->
+						<a ng-click="goToSection('.campuses, 'campuses')"> Campuses </a>
```

Comment:
I thought in this case if I used single quotes on the outside of the function it would mess up the single quotes inside because then it would be 
`'gotToSection('.what-is')'`
and so it would get red as `goToSection(` is a string and `)` is another string. 

That's why I always use single quotes first.